website/content/docs/auth/jwt/oidc-providers/index.mdx

```diff
@@ -8,12 +8,12 @@ description: OIDC provider configuration quick starts
 
 This page collects high-level setup steps on how to configure an OIDC
 application for various providers. For more general usage and operation
-information, see the [Vault JWT/OIDC method documentation](/vault/docs/auth/jwt).
+information, see the [OpenBao JWT/OIDC method documentation](/vault/docs/auth/jwt).
 
 OIDC providers are often highly configurable, and you should become familiar with
 their recommended settings and best practices. The guides listed below are
 largely community-driven and intended to help you get started. Corrections
-and additions may be submitted via the [Vault Github repository](https://github.com/hashicorp/vault).
+and additions may be submitted via the [OpenBao Github repository](https://github.com/hashicorp/openbao).
```

Comment:
This repo path is incorrect.  It should be openbao/openbao instead of hashicorp/openbao

Reply:
Good catch! I'll make sure to fix the repo references. 👍

Reply:
But since this PR is already approved, I'll open a new one for it once this is merged